# Tray contents message
# This structure contains the information of the current contents of a tray.

# The ID of the tray
string kit_tray

PublishedMovableTray movable_tray

# Collection of parts in the movable_tray frame
DetectedProduct[] products
